libs/cvr-fixture-generator/package.json

```diff
@@ -26,7 +26,6 @@
     "@votingworks/auth": "workspace:*",
     "@votingworks/backend": "workspace:*",
     "@votingworks/basics": "workspace:*",
-    "@votingworks/converter-nh-accuvote": "workspace:*",
```

Comment:
Same question here as above but for `libs/cvr-fixture-generator`

Reply:
We used to generate ballot page layouts in `converter-nh-accuvote` (see 5d64f735cf30efcb5e31e01a6f75058944be8560). It looks like we then removed the reference in 88836d906c2de65b149b78da28a087bc34897052.